src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue25089.cs

```diff
@@ -0,0 +1,29 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+    public class Issue25089 : _IssuesUITest
+    {
+		public Issue25089(TestDevice device): base(device)
+		{
+		}
+
+		public override string Issue => "OnAppearing of Page called again, although this page was on already replaced NavigationStack";
+
+		[Test]
+		[Category(UITestCategories.Shell)]
+		public void VerifyPageAppearingSequence()
```

Comment:
This test is failing on Android:
```
System.NullReferenceException : Object reference not set to an instance of an object.

 at UITest.Appium.HelperExtensions.GetText(IUIElement element) in /_/src/TestUtils/src/UITest.Appium/HelperExtensions.cs:line 72
   at Microsoft.Maui.TestCases.Tests.Issues.Issue25089.VerifyPageAppearingSequence() in /_/src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue25089.cs:line 26
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Void** arguments, Signature sig, Boolean isConstructor)
   at System.Reflection.MethodBaseInvoker.InvokeWithNoArgs(Object obj, BindingFlags invokeAttr)
```

Could you review it?

Reply:
@jsuarezruiz, I have resolved the test case failure on Android. Could you please validate it once and let me know if there are any concerns.